Tests/UnitTests/Core/EventData/CorrectedTransformFreeToBoundTests.cpp

```diff
@@ -91,7 +91,7 @@ BOOST_AUTO_TEST_CASE(CorrectedFreeToBoundTrackParameters) {
 
   // the jacobian from local to global at the starting position
   BoundToFreeMatrix boundToFreeJac =
-      eSurface->boundToFreeJacobian(geoCtx, eFreeParams);
+      eSurface->boundToFreeJacobian(geoCtx, tpos, dir);
```

Comment:
do we need the free parameter vector construction at all in this test?

```
  // construct the free parameters vector
  FreeVector eFreeParams = FreeVector::Zero();
  eFreeParams.segment<3>(eFreePos0) = tpos;
  eFreeParams[eFreeTime] = t;
  eFreeParams.segment<3>(eFreeDir0) = dir;
eFreeParams[eFreeQOverP] = qOverP;
```

Reply:
they are still in use a bit further down
```c++
  auto correctedRes = transformer(eFreeParams, eFreeCov, *eSurface, geoCtx);
```